app/controllers/api/integrations_controller.rb

```diff
@@ -4,6 +4,7 @@ class Api::IntegrationsController < Api::BaseController
   before_action :find_service
   before_action :find_proxy
   before_action :authorize
+  before_action :unauthorize_apiap, only: %i[edit update]
```

Comment:
```suggestion
  before_action :unauthorize_apiap, only: :edit
```

Reply:
I think we still need the update.

Reply:
This will happen with the "Promote" buttons if we unauthorize the `update` action:

<img width="1188" alt="Screen Shot 2019-10-08 at 10 29 38 AM" src="https://user-images.githubusercontent.com/1842261/66379845-8f5bf980-e9b6-11e9-90fc-7440d2ae837e.png">